index.html

```diff
@@ -7,6 +7,6 @@
 <title>Welcome to GitTraining</title>
     </head>
   <body>
-      
+     <h1> Welcome to my git training..i am just trying to play around. Stay blessed </h1> 
```

Comment:
can you remove the "I am just trying to play around " and make the Stay blessed  red colour

Reply:
yes i will